src/cpp/rtps/builtin/discovery/participant/PDP.cpp

```diff
@@ -1698,6 +1700,123 @@ void PDP::add_builtin_security_attributes(
 
 #endif // HAVE_SECURITY
 
+void PDP::local_participant_attributes_update_nts(
```

Comment:
As long as the method only updates locators and userData values, and not the remaining participant attributes, this method name may generate confusion.
What do you think about renaming it to be more specific?